### ISSUE TITLE
The type or namespace name 'Design' does not exist in the namespace 'ComponentFactory.Krypton

### ISSUE THREAD

**Issue by xiaoqianshijian** [ComponentFactory/Krypton#141]
I compile the binaries， but it prompt "The type or namespace name 'Design' does not exist in the namespace 'ComponentFactory.Krypton'";

**Comment by jpix** [comment]
Me too

**Comment by Wagnerp** [comment]
@jpix 

You can't compile this repo for 4.7, as this is only for 2.0. You need to head [here](https://github.com/Wagnerp/Krypton-NET-4.70) for the 4.7 version, and it works with VS 2017.

**Comment by jpix** [comment]
Thank you! It worked.

**Comment by xiaoqianshijian** [comment]
@Wagnerp 
I want to compile krypton for .Net 4.0 and work with VS 2015, is there for the 4.0 version?

**Comment by Wagnerp** [comment]
@xiaoqianshijian 

Yes there is [here](https://github.com/Wagnerp/Krypton-NET-4.00). It's still a VS 2017 solution, however you could always import the `*.csproj` files from it into a empty VS 2015 solution.

**Comment by Wagnerp** [comment]
@jpix 

Your welcome, the bonus with all our repositories from 4.00 to 4.80, is that we've ripped out the `Designer.dll`, so all components will work out-of-the-box with no funky setup required!
